docs/_docs/installation.md

```diff
@@ -10,16 +10,31 @@ encountered and how we might make the process easier.
 
 ### Requirements
 
-Installing Jekyll is easy and straight-forward, but there are a few
-requirements you’ll need to make sure your system has before you start.
+Installing Jekyll ought to be straight-forward if all requirements are met.
+Before you start, make sure your system has the following:
 
-- [Ruby](https://www.ruby-lang.org/en/downloads/) (including development
+- Linux, Unix, or macOS
+- [Ruby](https://www.ruby-lang.org/en/downloads/) (including all development
   headers, v1.9.3 or above for Jekyll 2 and v2 or above for Jekyll 3)
 - [RubyGems](https://rubygems.org/pages/download)
-- Linux, Unix, or macOS
-- [NodeJS](https://nodejs.org/), or another JavaScript runtime (Jekyll 2 and
-earlier, for CoffeeScript support).
-- [Python 2.7](https://www.python.org/downloads/) (for Jekyll 2 and earlier)
+
+#### Only required for Jekyll 2 and earlier
+- [NodeJS](https://nodejs.org/), or another JavaScript runtime (for CoffeeScript support).
+- [Python 2.7](https://www.python.org/downloads/)
+
+<div class="note info">
+  <h5>Running Jekyll on Ubuntu</h5>
+  <p>
+    Users of Jekyll on Ubuntu have reported encountering 
+    <i>Could not locate Gemfile or .bundle/ directory</i> error messages at the 
+    <code>bundle exec jekyll serve</code> step in the <a href="../quickstart/">Quick-start guide</a>.
```

Comment:
When I tried to reproduce this issue, I could not get past the `gem install jekyll bundler` step. It gave me a clear message that it was unable to install **ffi** because the Ruby development headers were missing.

With this in mind, could we make this section just a bit more generic? This this has caused problems for me on _macOS_ as well, so it's not just an Ubuntu thing.